modules/oe_translation_epoetry/src/RequestFactory.php

```diff
@@ -259,7 +259,7 @@ protected function toRequestDetails(TranslationRequestEpoetryInterface $request)
     $content = $this->formatter->export($request);
     $original_document = (new OriginalDocumentIn())
       ->setTrackChanges(FALSE)
-      ->setFileName($entity->label())
+      ->setFileName(str_replace(' ', '-', $entity->label()) . '.html')
```

Comment:
I updated the file name in the request as spaces cannot exist and they want also an extension.